src/test/ui/unboxed-closures/unboxed-closures-infer-fnonce-move-call-twice.stderr

```diff
@@ -16,6 +16,10 @@ note: this value implements `FnOnce`, which causes it to be moved when called
    |
 LL |     tick();
    |     ^^^^
+help: consider cloning the value if the performance cost is acceptable
+   |
+LL |     tick.clone()();
+   |         ++++++++
```

Comment:
This is unfortunate.

Reply:
Yeah, I guess by this point in MIR we can't see this is an expr being called?